public/static/partials/project_trigger_modal.html

```diff
@@ -0,0 +1,65 @@
+<md-dialog style="width:650px">
+    <md-dialog-content>
+        <form ngSubmit="addTrigger()">
+            <md-card style="width:97.5%">
+                <md-card-title>
+                    <div ng-bind="modalTitle()"></div>
+                </md-card-title>
+                <md-card-content>
+                    <section layout="row" flex>
+                        <md-input-container flex=50>
+                            <label>Project:</label>
+                            <input type="text" ng-model="trigger.project" ng-required="true">
+                        </md-input-container>
+                        <md-input-container flex=50>
+                            <label>Date cutoff:</label>
```

Comment:
I feel like we should use a different term than date cutoff? let's think/poll some people on this.